### PR TITLE
removed the restyling of the h tags in the about section

### PR DESCRIPTION
**Pull request by jonhartzler**
Not necessary since the example uses an h5 now instead of an h3.

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

**Comment by jaronkk** [comment]
:+1:

File: app/assets/stylesheets/ui-customizations.css.scss [hunk 1]
```diff
@@ -247,7 +247,6 @@ body .jumbotron, .container .jumbotron, .container-fluid .jumbotron {
   text-align: left;
 
   .header-position {
-    position: absolute;
```
Comment (jaronkk) [inline app/assets/stylesheets/ui-customizations.css.scss]: Why was this removed?

Comment (jonhartzler) [inline app/assets/stylesheets/ui-customizations.css.scss]: because i hit cmd z twice.   It is part of another pull request I will remove.

**Comment by jaronkk** [comment]
@jonhartzler I rebased and squashed the two commits together.
